src/index.ts

```diff
@@ -45,11 +45,20 @@ async function main() {
   });
 
   if (workflow_id) {
-    // The user provided one or more workflow id
-    workflow_id
-      .replace(/\s/g, '')
-      .split(',')
-      .forEach(n => workflow_ids.push(n));
+    // When workflow_id is 'all'
+    // Ids of the all workflows be taken to cancel
+    if (workflow_id === 'all') {
+      const {
+        data: { workflows: allWorkflows },
+      } = await octokit.rest.actions.listRepoWorkflows({ owner, repo });
```

Comment:
This only lists owner and repo but don't you need to pass the ref/branch?

Reply:
@styfle Apologies for the delay, GitHub docs is intermittently down in the last 1 week, but I somehow got hold of the documentation in Github. The function `listRepoWorkflows` only accepts owner & repo there is no parameter for branch